db/seeds.rb

```diff
@@ -219,7 +224,7 @@ def add_allergens_and_diets(item)
 
 # Creates users, each user will have an ingredient to give.
 @counter_from_zero = 0
-26.times do
+25.times do
```

Comment:
1 less so Zucchini is reserved for me during the demo. I will add it and Williams will request it,.